FordLabs/PeopleMover
# Vertically center text on space tile

## Issue
Connects #380

## What was done
- [x] Vertically center text on space tile

## How to test
1. Go to the the dashboard and create a space with all 40 characters and all W's ( `WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW` ) and ensure that the name and last updated date is all contained in the space tile and vertically centered.
